compiler/AST/build.cpp

```diff
@@ -2185,6 +2185,180 @@ BlockStmt* buildLocalStmt(Expr* condExpr, Expr *stmt) {
       buildLocalStmt(stmt->copy()), stmt);
 }
 
+/*
+  Builds the try/catch part of the manager block:
+
+  try {
+    // Insertion point for next manager or user block.
+  } catch chpl_tmp_err {
+    errorCaught = true;
+    manager.leaveThis(chpl_tmp_err);
+  }
+
+*/
+static TryStmt* buildTryCatchForManagerBlock(VarSymbol* managerHandle,
+                                             VarSymbol* errorCaught) {
+  const char* errName = "chpl_tmp_err";
+
+  // Build the catch block.
+  auto catchBlock = new BlockStmt();
+
+  // BUILD: errorCaught = true;
+  auto seErrorCaught = new SymExpr(errorCaught);
+  auto seTrue = new SymExpr(new_BoolSymbol(true));
```

Comment:
Maybe use `gTrue`?

Reply:
Oh that works? Thanks!

Reply:
seems this change is still todo. BTW you should be able to just pass `gTrue` as the argument to the CallExpr constructor (it should add a SymExpr in that case).